tests/vmipreset_test.go

```diff
@@ -459,6 +459,9 @@ var _ = Describe("[rfe_id:609][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			err := virtClient.RestClient().Post().Resource("virtualmachineinstancepresets").Namespace(tests.NamespaceTestDefault).Body(preset).Do().Error()
 			Expect(err).ToNot(HaveOccurred())
 
+			// Give virt-api's cache time to sync before proceeding
+			time.Sleep(3 * time.Second)
```

Comment:
I think a retry loop is a better approach, because it is less fragile (will 3 seconds always be enough?) and, in presence of failures, gives us a better picture of what went wrong.
Unfortunately I can't suggest a more robust way.

Reply:
The problem is we have zero visibility into what to retry. :/ We can't query virt-api's informer cache directly.